wiki/Article_Style_Guide/en.md

```diff
@@ -68,11 +68,11 @@ If your language isn't listed below, please open an issue in the `osu-web` repos
 - `it.md`: Italian
 - `ja.md`: Japanese
 - `ko.md`: Korean
-- `nl.md`: Nederlands
+- `nl.md`: Dutch
 - `no.md`: Norwegian
 - `pl.md`: Polish
-- `pt.md`: Português
-- `pt-br.md`: Português (Brasil)
+- `pt.md`: Portuguese
+- `pt-br.md`: Portuguese (Brasil)
```

Comment:
`Brazilian Portuguese` or `Portuguese (Brazil)`, but Brazilian Portuguese is prefered.